cached-tables.md

```diff
@@ -0,0 +1,252 @@
+---
+title: Cached Tables
+summary: Learn the cached table feature in TiDB, which is used for rarely-updated small hotspot tables to improve read performance.
+---
+
+# Cached Tables
+
+In v6.0.0, TiDB introduces the cached table feature for frequently accessed but rarely updated small hotspot tables. When this feature is used, the data of an entire table is loaded into the memory of the TiDB server, and TiDB directly gets the table data from the memory without accessing TiKV, which improves the read performance.
+
+This document describes the usage scenarios of cached tables, the examples, and the compatibility restrictions with other TiDB features.
+
+## Usage scenarios
+
+The cached table feature is suitable for tables with the following characteristics:
+
+- The data volume of the table is small.
+- The table is read-only or rarely updated.
+- The table is frequently accessed, and you expect a better read performance.
+
+When the data volume of the table is small but the data is frequently accessed, the data is concentrated on a Region in TiKV and makes it a hotspot Region, which affects the performance. Therefore, the typical usage scenarios of cached tables are as follows:
```

Comment:
```suggestion
When the data volume of the table is small but the data is frequently accessed, the data is concentrated on a Region in TiKV and makes it a hotspot Region, which affects the performance of TiKV. Therefore, the typical usage scenarios of cached tables are as follows:
```

Reply:
Not sure the performance of TiKV or of multiple components as a whole.